src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,10 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
-    public String getRandomColor() {
-        return null;
+    public Color getRandomColor() {
+        int index = new Random().nextInt(Color.values().length);
```

Comment:
![image](https://github.com/mate-academy/jv-lottery/assets/20326858/4b1d3eb1-2f8d-4e3c-aabc-cb4652a13e9b)

Reply:
not fixed